integration-tests/sceptre-project/config/9/B.yaml

```diff
@@ -1,5 +0,0 @@
-template:
```

Comment:
I've renamed this StackGroup to "external-stack-output" because "9" was never a good name for a test fixture. And I've renamed this file to "resolver-with-profile-and-region" to properly describe what was being tested.